frontend/src/metabase/writeback/components/ActionCreator/ActionCreator.tsx

```diff
@@ -88,4 +108,7 @@ function ActionCreatorComponent({
   );
 }
 
-export const ActionCreator = connect(mapStateToProps)(ActionCreatorComponent);
+export const ActionCreator = _.compose(
+  Actions.load({ id: (state: State, props: any) => props.actionId }),
```

Comment:
FYI, if you want to avoid `any` here and have a bit more type-safety, you can break down the `Props` type a bit. See how it's done [here](https://github.com/metabase/metabase/blob/cf10a9d33adec9f6f9f37c779803cb29af11afd8/frontend/src/metabase/admin/tasks/containers/ModelCacheRefreshJobs/ModelCacheRefreshJobModal.tsx#L90) for instance